nextcord/types/embed.py

```diff
@@ -0,0 +1,179 @@
+"""
+The MIT License (MIT)
```

Comment:
I honestly dislike the embed api, it looks super messy imo

Reply:
> I honestly dislike the embed api, it looks super messy imo

Oh? Why that? Because of a class per Proxy?